activities/procurement/principles.md

```diff
@@ -14,3 +14,12 @@ This means the services and products we use need to be:
 * sustainable for the organization as well as society and the planet
 
 The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md).
+
+## Procurement process
+
+To make sure the procurement process is fair, we adhere to the following requirements:
+
+* we request 3 - 4 quotes from different providers
+* check if prices are competitive and in line with the market
+* if products are sustainable and / or local
+* final decision taken with operations team
```

Comment:
The order in this list is important so it should be an ordered list.

Ordered lists have numbers in front of them, either incrementing or all the same. I recommend replacing `* ` with `1. `.